test/utility/parse_opts.c

```diff
@@ -158,7 +245,9 @@ testutil_parse_begin_opt(int argc, char *const *argv, const char *getopts_string
       USAGE_STR('C', " [-C]"), USAGE_STR('d', " [-d add data]"), USAGE_STR('h', " [-h home]"),
       USAGE_STR('m', " [-m]"), USAGE_STR('n', " [-n record count]"),
       USAGE_STR('o', " [-o op count]"),
-      USAGE_STR('P', " [-PT] [-Po storage source] [-PSD<data_seed>,E<extra_seed>]"),
+      USAGE_STR('P',
+        " [-PT] [-Po storage source] [-PSD<data_seed>,E<extra_seed>] [-Pd <force_delay><delay_ms>] "
+        "[-Pe <force_error><error_ms>]"),
```

Comment:
I'd put a comma between `<force_delay>` and `<delay_ms>`. Same for the other one.

Reply:
Also alphabetize the -P options in the usage.

Reply:
Fixed in 9e36aaa699dcd933cf4d399d2dea888ed80ae3ba